src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue21916.cs

```diff
@@ -0,0 +1,28 @@
+#if !ANDROID && !MACCATALYST
```

Comment:
Why avoid Android? Could be nice to test the same behavior in all the platforms.

Reply:
On the Android platform, the Unloaded event of the current page is not triggered when navigating to the next page. As a result, the UI test case for Android has been skipped. Let me know if you have any further concerns. 

Android issue link : https://github.com/dotnet/maui/issues/26050

Reply:
@jsuarezruiz , here is the link to the issue - https://github.com/dotnet/maui/issues/26050